runtime/test/data-layer-test.js

```diff
@@ -41,6 +41,8 @@ describe('entity', async function() {
 
 describe.skip('relation', function() {
   it('can be created, stored, and restored', function() {
+    let Relation;
```

Comment:
Won't line 46/47 (`new Relation`) always fail because `Relation` is undefined? This may be a misunderstanding of JS on my part though :)

Reply:
It would fail, but this test is skipped. Most of the code here has evolved significantly since it was written.